docker/Dockerfile

```diff
@@ -65,68 +65,6 @@ RUN apt-get update \
 RUN pip3 install setuptools
 
 
-# -------------=== Build openssl_image ===-------------
```

Comment:
Does verify report in Avalon shell use openSSL. @manju956 @pankajgoyal2  can you please confirm if verify report will work if we remove openSSL (untrusted) from Avalon shell docker image

Reply:
Generic client uses it to verify the AVR report of worker.

Reply:
Verify_report module is built as separate python package. It depends on openssl as client invokes untrusted cpp code to do verification of attestation data.